libs/gl-client/src/signer/mod.rs

```diff
@@ -63,8 +68,14 @@ pub enum Error {
     #[error("scheduler returned faulty URI: {0}")]
     InvalidUri(#[from] InvalidUri),
 
-    #[error("signer refused a request: {0}")]
-    Signer(#[from] vls_protocol::Error),
+    #[error("resolver error: request {0:?}, context: {1:?}")]
```

Comment:
I like this! I am far to sloppy, polluting everything with `anyhow!` errors. It is so much better to keep possible errors at one point in the file to find them!